src/pages/EditMonitor.vue

```diff
@@ -620,13 +620,23 @@ export default {
         },
 
         pageName() {
-            return this.$t((this.isAdd) ? "Add New Monitor" : "Edit");
+            let name = "Add New Monitor";
+            if (this.isClone) {
+                name = "Clone Monitor";
+            } else if (this.isEdit) {
+                name = "Edit";
+            }
+            return this.$t(name);
```

Comment:
I would do as:
```Javascript
let name;
if (this.isClone) {
  name = "Clone Monitor";
} else if (this.isAdd) {
  name = "Edit";
} else if (this.isEdit) {
  name = "Add New Monitor";
} else {
	name = "Wrong Monitor Action";
}
return this.$t(name);
```

And I would add in en.js: Wrong Monitor Action

Also, it's not quite clear to me why the Edit Monitor translation key wouldn't be added, so that when you change the Edit key for other purposes, you don't risk making this one case incomprehensible

Reply:
Mixed the name value for isAdd and isEdit?
Adding translations would be good yes, I just did as little as possible